nym-vpn-core/crates/nym-vpnd/src/command_interface/listener.rs

```diff
@@ -895,6 +896,18 @@ impl NymVpnd for CommandInterface {
 
         Ok(tonic::Response::new(response))
     }
+
+    async fn get_log_path(
+        &self,
+        _: tonic::Request<()>,
+    ) -> Result<tonic::Response<GetLogPathResponse>, tonic::Status> {
+        let dir = service::log_dir();
```

Comment:
I can see this being fragile in the future. Ideally I think we should in the running instance track the log path similar to how we track the data path, so that we return the log dir used (output), not the input

Maybe you could try adding this to `LoggingSetup`?

Reply:
Daemon architecture is quite complex
I'll try to improve this in a followup PR